### ISSUE TITLE
GUI map creation

### ISSUE THREAD

**Issue by nskins** [nskins/goby#82]
Map creation would be much faster with a GUI.

**Comment by FatihDurmus** [comment]
can i try this quest ?

**Comment by nskins** [comment]
@FatihDurmus Yes, please! Although, I must admit, I don't have many ideas about the implementation. Let me know if I can help or do code review.

**Comment by tston529** [comment]
I have no idea if anyone is still working on this project, but I have a soft spot for text based rpgs, so I started working on a map GUI over the weekend.  It's Python, but feel free to give me a hand, since I can't work on it too much during the work week.
https://github.com/tston529/goby-map-editor
